.github/workflows/scheduled_releases.yml

```
@@ -25,7 +25,7 @@ jobs:
       matrix:
        # we support the trailing four versions plus the latest pre-release
        # and the upcoming version's pre-release.
-       version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0b2"]
+       version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0"]
```

We will still want to include pre-release 1.6 for now
```suggestion
       version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0b2", "1.6.0"]
```